### PR TITLE
Fixing IMU frame_id issue

### PR DESCRIPTION
**Pull request by ayrton04**
For issue #3. Note that I am merging into the other fix. @bradleypowers @kayak0806 @davelkan @tappan-at-git

### CODE REVIEW

File: gazebo_plugins/src/gazebo_ros_imu.cpp [hunk 1]
```diff
@@ -99,6 +99,15 @@ void GazeboRosIMU::LoadThread()
...
     this->link_name_ = this->sdf->Get<std::string>("bodyName");
 
+  if (!this->sdf->HasElement("frameId"))
+  {
+    ROS_INFO("imu plugin missing <frameId>, defaults to 'imu'");
+    this->frame_id_ = "imu";
+    return;
+  }
+  else
+    this->frame_id_ = this->sdf->Get<std::string>("frameId");
```
Comment (tappan-at-git) [inline gazebo_plugins/src/gazebo_ros_imu.cpp:109]: Personal preference, but could we wrap the else block in curly braces? Obviously it's syntactically fine either way, I just hate forgetting to add the braces when I want to add more lines to a block.

**Comment by tappan-at-git** [comment]
One small stylistic thing, otherwise looks good to me.

File: gazebo_plugins/src/gazebo_ros_imu.cpp (hunk 1 above)
Comment (ayrton04) [inline gazebo_plugins/src/gazebo_ros_imu.cpp:109]: So I am a HUGE stickler for _always_ using curly braces. The only reason they are not here is to keep the style in line with the upstream code (see the other parameters being loaded), in the event that we ever push to it.

**Comment by bradleypowers** [comment]
:+1: Merging.

**Pull request merged**